src/test/java/com/programmers/repository/TestBookRepositoryTest.java

```diff
@@ -0,0 +1,93 @@
+package com.programmers.repository;
+
+import com.programmers.domain.Book;
+import com.programmers.front.BookConsole;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.*;
+
+
+class TestBookRepositoryTest {
+
+    private final BookRepository bookRepository = new TestBookRepository();
+
+
+    @BeforeEach
+    public void setUp(){
+        bookRepository.saveBook(new Book("test1", "author", 344));
+        bookRepository.saveBook(new Book("test21", "author", 344));
+        bookRepository.saveBook(new Book("test312", "author", 344));
+    }
+
+    @AfterEach
+    public void clear(){
+        bookRepository.clear();
+    }
+
+
+    @Test
+    @DisplayName("Repository: 정상적으로 책 등록하는 경우")
+    void enrollBook() throws Exception {
+        //given
+        Book book = new Book("enrollTest", "testAuthor", 122);
+        //when
+        bookRepository.saveBook(book);
+        List<Book> list = bookRepository.findAll();
+        Book findBook = list.get(list.size() - 1);
+
+        //then
+        assertThat(findBook.getBookId()).isEqualTo(Long.valueOf(list.size()));
+        assertThat(findBook.getBookId()).isEqualTo(4L);
+        assertThat(book).isEqualTo(findBook);
+    }
+
+    @Test
+    @DisplayName("Repository: 책을 정상적으로 모두 출력하는 경우 테스트")
+    public void findAllBookRepositoryTest() throws Exception {
+        //given
+        List<Book> bookList = bookRepository.findAll();
+        Assertions.assertThat(bookList.size()).isEqualTo(3);
+
+        //when
+//        BookConsole.showAllBooks(bookList);
+        Book book = new Book("enrollTest", "testAuthor", 122);
+        bookRepository.saveBook(book);
+
+        bookList = bookRepository.findAll();
+
+        //then
+        Assertions.assertThat(bookList.size()).isEqualTo(4);
```

Comment:
```suggestion
        assertThat(bookList).hasSize(4);
```

사이즈를 검증하는 assertJ 표현법을 활용해보시죠

Reply:
[[refactor] 1. 테스트 코드 컬렉션 사이즈 비교 메서드 변경](https://github.com/prgrms-be-devcourse/java-library-management/commit/163babd87385dc4619bd8222ca36ca3ab08af260)

반영 완료했습니다!